data/zilliqa/assets/zil1gcdkmkuunj0l6hjl48zzye50g397dgslzdf4uk/meta.json

```diff
@@ -0,0 +1,15 @@
+{
+	"name": "Oasis of Amazing Creatures",
+	"symbol": "OAC",
+	"web": "https://sites.google.com/view/oasis-of-amazing-creatures/home",
+	"links": {
+		"Github": "oasisofamazing",
+		"Twitter": "oasisofamazing"
```

Comment:
twitter should be lowercase

Reply:
Thanks. We have changed it to "twitter": "oasisofamazing"